react-backend/database/queries/getUsersFilteredBy.js

```diff
@@ -0,0 +1,99 @@
+const dbConnection = require('../dbConnection');
+
+const query = (filterBy, orderBy) => `
+SELECT users.id, users.fullName, users.sex, users.yearofbirth, users.email, users.date
+FROM users
+WHERE users.cb_id = $1 ${filterBy} ${orderBy}`;
+
+const arrayMatch = (arr, search) => {
+  return arr.some(el => el === search);
+};
+
+const getFiltersQuery = filterBy => {
```

Comment:
As mentioned in [this comment](https://github.com/FAC-11/DataPower/pull/151#discussion_r161210085), I'll have a go at refactoring this but merge for now :+1: